mne/preprocessing/tests/test_xdawn.py

```diff
@@ -136,6 +136,9 @@ def test_xdawn_regularization():
     modified_event = events[sel[0]]
     modified_event[0] += 1
     epochs.events[sel[1]] = modified_event
+    sel = np.where(events[:, 2] == 2)[0][:2]
+    epochs.events[sel[1], 0] = events[sel[1], 0] + 1
+    epochs.events[sel[1], 2] = events[sel[1], 2]
```

Comment:
I think it is basically moves the start of one event to the start of the next event + 1. This aims at creating an overlap.